### DISCUSSION TITLE
Color issus in Django Admin dark mode

### DISCUSSION THREAD

**Discussion by jonocodes** [codingjoe/django-select2#284] [Q&A]
### Description

When using dark mode that comes with django admin, the colors make the widget unreadable.

In this screenshot:
* Regions is using forms.MultipleChoiceField
* Operators is using forms.ChoiceField(widget=Select2MultipleWidge

![Screen Shot 2024-06-01 at 09 57 22](https://github.com/codingjoe/django-select2/assets/1310468/1a6c3a31-cfed-477d-8d14-fb0b7aa3041d)

### Steps to Reproduce

1. Create a Select2 field. In my case I am using `forms.ChoiceField(widget=Select2MultipleWidget, choices=get_sorted_operator_choices())`
2. Render the page and see that the text is white in a white box.

Note: I am not using the built in admin autoselect since these are not foreign keys. It is a foreign table, but using a different field then the foreign key.

I'm using vanilla Django 5.x. I believe dark mode was introduced at 3.2.

### Expected Behavior

Like other dark mode select fields, the field should be black and the text white.

**Comment by codingjoe** [comment]
Hi there 👋,

Thank you for reaching out. You will have to use the `Select2AdminMixin`, see also: https://github.com/codingjoe/django-select2/blob/f427b3113d47cf60ab524e9c32cbd336e01417c5/django_select2/forms.py#L163-L175

Cheers!
Joe
